dispatchers/Html.php

```diff
@@ -154,7 +154,10 @@ public function enqueue_assets() {
 		wp_enqueue_script(
 			'query-monitor',
 			$this->qm->plugin_url( 'assets/query-monitor.js' ),
-			array( 'jquery' ),
+			array(
+				'jquery',
+				'wp-util',
```

Comment:
I think you should either add `underscore` or remove `jquery`, as `wp-util` depends on both.

Reply:
I think I'll add Underscore so the dependencies are more explicit.